libr/anal/p/anal_arm_cs.c

```diff
@@ -58,6 +58,9 @@
 #define ISPREINDEX64() ((OPCOUNT64() == 3) && (ISMEM64(2)) && (ISWRITEBACK64()))
 #define ISPOSTINDEX64() ((OPCOUNT64() == 4) && (ISIMM64(3)) && (ISWRITEBACK64()))
 
+static RRegItem base_regs[4];
```

Comment:
avoid globals

Reply:
OK, I'll change that.

Reply:
Is there any better way to do that other than dynamically allocating op regs? I don't want to do that this way, because it won't be freed by r_anal_op_fini (it's TODO in r_anal_value_free). Of course, if you wish, I may fix it, so it frees also RRegItems and implement it this way.